src/main/java/sparksoniq/jsoniq/runtime/iterator/functions/base/Functions.java

```diff
@@ -21,15 +21,17 @@
 
 import sparksoniq.exceptions.UnknownFunctionCallException;
 import sparksoniq.jsoniq.compiler.translator.expr.primary.FunctionCall;
-import sparksoniq.jsoniq.item.IntegerItem;
 import sparksoniq.jsoniq.runtime.iterator.RuntimeIterator;
-import sparksoniq.jsoniq.runtime.iterator.functions.*;
-import sparksoniq.jsoniq.runtime.iterator.functions.arithmetic.AvgFunctionIterator;
-import sparksoniq.jsoniq.runtime.iterator.functions.arithmetic.MaxFunctionIterator;
-import sparksoniq.jsoniq.runtime.iterator.functions.arithmetic.MinFunctionIterator;
-import sparksoniq.jsoniq.runtime.iterator.functions.arithmetic.SumFunctionIterator;
+import sparksoniq.jsoniq.runtime.iterator.functions.arrays.ArraySizeFunctionIterator;
+import sparksoniq.jsoniq.runtime.iterator.functions.sequences.aggregate.*;
```

Comment:
The * imports are in general, to be avoided but we can think about that later. I also forgot about them.